pkg/operator2/starter.go

```diff
@@ -127,6 +139,20 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return err
 	}
 
+	// add syncing for etcd certs for oauthapi-server
+	if err := resourceSyncer.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: "openshift-oauth-apiserver", Name: "etcd-serving-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: "openshift-config", Name: "etcd-serving-ca"},
+	); err != nil {
+		return err
+	}
+	if err := resourceSyncer.SyncSecret(
+		resourcesynccontroller.ResourceLocation{Namespace: "openshift-oauth-apiserver", Name: "etcd-client"},
+		resourcesynccontroller.ResourceLocation{Namespace: "openshift-config", Name: "etcd-client"},
+	); err != nil {
+		return err
+	}
+
```

Comment:
I wonder whether we can split the starter according to operand a bit more. Would make it easier to read.